modules/gdscript/gdscript_editor.cpp

```diff
@@ -403,6 +403,11 @@ void GDScriptLanguage::debug_get_globals(List<String> *p_globals, List<Variant>
 
 		const Variant &var = gl_array[E.value];
 		if (Object *obj = var) {
```

Comment:
```suggestion
		bool freed = false;
		if (Object *obj = var.get_validated_object_with_check(freed)) {
```
Instead, no need to complicate things

Reply:
Done!